### ISSUE TITLE
Update plugin to use new cocoa pod SAMKeychain instead of obsoleted SSKeychain

### ISSUE THREAD

**Issue by AntonioCuevaUrraco** [hamorphis/nativescript-ios-uuid#3]
Any app using this plugin will most probably crash if using the plugin. I have been checking with the beta and it crash on the line.

``` js
var strApplicationUUID = SSKeychain.passwordForServiceAccount(appName, "incoding");
```

If you change the dependency to SAMKeychain as they suggest because SSKeychain is deprecated it stop crashing but it doesn't work anyway as SAMKeychain is not able to retrieve the token, and generates a new one every time.

It seams that something as simple as to get an unique ID is pretty complicated on ios nowadays :-(

**Comment by roblav96** [comment]
@AntonioCuevaUrraco have a look here https://github.com/fabiocaccamo/FCUUID

**Comment by hamorphis** [comment]
@AntonioCuevaUrraco I will have to update the plugin and test. I am not sure.

**Comment by hamorphis** [comment]
@AntonioCuevaUrraco @roblav96 I released v1.0.0: https://www.npmjs.com/package/nativescript-ios-uuid. Please, give it a try and let me know if something does not work properly.
